### PR TITLE
Doc: Update Files metadata table

### PR DESCRIPTION
**Pull request by szehon-ho**
Files metadata table was missing some columns, and also missing an example for partitioned tables in which the new partition column is added (users may want to know how to check what data files they have for a given partition)

### CODE REVIEW

**Comment by kbendick** [comment]
cc @samredai as the docs refactor is underway 👍

File: site/docs/spark-queries.md [hunk 1]
```diff
@@ -209,16 +209,30 @@ To show a table's data files and each file's metadata, run:
...
-| s3:/.../table/data/00001-4-8d6d60e8-d427-4809-bcf0-f5d45a4aad96.parquet | PARQUET     | 1            | 597                | [1 -> 90, 2 -> 62] | [1 -> 1, 2 -> 1] | [1 -> 0, 2 -> 0]  | []               | [1 -> , 2 -> b] | [1 -> , 2 -> b] | null         | [4]           |
-| s3:/.../table/data/00002-5-8d6d60e8-d427-4809-bcf0-f5d45a4aad96.parquet | PARQUET     | 1            | 597                | [1 -> 90, 2 -> 62] | [1 -> 1, 2 -> 1] | [1 -> 0, 2 -> 0]  | []               | [1 -> , 2 -> a] | [1 -> , 2 -> a] | null         | [4]           |
-+-------------------------------------------------------------------------+-------------+--------------+--------------------+--------------------+------------------+-------------------+------------------+-----------------+-----------------+--------------+---------------+
++-------+-------------------------------------------------------------------------+-----------+---------------+------------+------------------+---------------------------+------------------------+------------------------+----------------+---------------------------------------+---------------------------------------+------------+-------------+------------+-------------+
```
Comment (rdblue) [inline site/docs/spark-queries.md:215]: Is it possible to not revert the formatting changes? I think it is less readable with the initial space removed.

Maybe we should replace this with a real HTML table?

Here's a snippet of code that we use in our notebooks to format PySpark dataframes as nicer tables:

```python
from prettytable import PrettyTable
from IPython.core.magic import register_line_cell_magic

class DFTable(PrettyTable):
    def __repr__(self):
        return self.get_string()

    def _repr_html_(self):
        return self.get_html_string()

def to_table(df, num_rows=100):
    cols = df.columns

    t = DFTable()
    t.field_names = cols
    t.align = "r"
    for row in df.limit(num_rows).collect():
        d = row.asDict()
        t.add_row([ d[col] for col in cols ])

    return t
```

That will produce both HTML and text tables that have reasonable formatting.

**Review by kbendick — COMMENTED** [review]
Somewhat unrelated, but I noticed that lower_bounds has no value for some keys. Is this possibly confusing for readers of the docs?

`[1 -> , 2 -> c] `

Notice that `1` doesn't point to anything. Is this intentional and do we think this might confuse readers? I can open a separate issue if so.

File: site/docs/spark-queries.md (hunk 1 above)
Comment (szehon-ho) [reply site/docs/spark-queries.md:215]: Makes sense, let me look at this.

**Comment by szehon-ho** [comment]
@rdblue @kbendick @samredai  i made a markdown table for files, if you guys preview the file in github can see what it looks like.  If you want i can extend this to the other ones too, or do it separately.

By the way i also found a possible problem in the spec_id column added in #3015 , it should probably get hidden just like partition column if table is unpartitioned, I could fix that in another PR.

**Comment by samredai** [comment]
GitHub injects some overflow css to add the scroll bar, if we switch to markdown tables we'd have to add that as well. Here's how the table currently would look:
Screen Shot 2021-11-05 at 9 22 44 PM

But adding the following:
### extra.css
```css
.markdown-table-container {
  width: 780px;
  overflow-x: scroll;
}
```
and then surrounding your markdown table with:
```html
<div class="markdown-table-container" markdown="block">
...markdown table here...
</div>
```
will make it look like this with the scrollbar:
Screen Shot 2021-11-05 at 9 44 05 PM

**Comment by rdblue** [comment]
+1 to adding the scroll bar. Thanks @samredai!

**Comment by rdblue** [comment]
The markdown table looks good to me. The only problem is that now just this table is markdown. Anyone want to follow up with an update for the other tables?

**Comment by KnightChess** [comment]
> The markdown table looks good to me. The only problem is that now just this table is markdown. Anyone want to follow up with an update for the other tables?

@rdblue I can update other tables in this pr #3482

**Comment by szehon-ho** [comment]
Screenshot 2021-11-08 at 21 22 30

Thanks @samredai and @rdblue  for review , added the scroll bar using div and css as suggested.  Attached screenshot

**Comment by KnightChess** [comment]
image
use scroll bar, there will be a empty line under the form when there is not overflow. did I not use it or optimize the scroll bar @szehon-ho

**Comment by szehon-ho** [comment]
@KnightChess  I'm not sure to be honest, can you put up your change on your pr?  We can continue discussion on that pr instead of this one?

**Comment by samredai** [comment]
@szehon-ho can you update the CSS to the below? If we're using this for all markdown tables, `auto` will hide the scrollbar space that @KnightChess is seeing when the table width is below the container width.
```css
.markdown-table-container {
  width: 780px;
  overflow-x: auto;
}
```

**Comment by szehon-ho** [comment]
Hey @samredai @rdblue , would we able to continue moving this forward?  Was out on vacation, but my understanding is that we still keep the markdown files in this repo?

**Comment by samredai** [comment]
@szehon-ho yes the plan is to keep the markdown files in this repo. There are some changes coming soon but any merged docs PR will be included. This looks good to me, I'll let @rdblue comment on if it's good to merge.

**Review by rdblue — APPROVED** [review]

**Comment by rdblue** [comment]
Merged. Thanks, @szehon-ho!

**Comment by szehon-ho** [comment]
Thanks for fast response !

**Pull request merged**